taxi/admin.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from taxi.models import Manufacturer, Car, Driver
+
+
+@admin.register(Manufacturer)
+class ManufacturerAdmin(admin.ModelAdmin):
+    list_display = ('name', 'country')
+    search_fields = ('name', 'country')
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ('license_number',)
+    fieldsets = UserAdmin.fieldsets + (
+        ('Additional info', {"fields": ('license_number',)}), )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ('Additional info', {"fields": ('license_number',)}), )
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ('model',)
+    list_display = ('manufacturer',)
```

Comment:
You might want to include the car model in the list display for the Car model. It would be more informative for the user. You can do this by changing the line to: `list_display = ('manufacturer', 'model')`